pkg/server/sandbox_stop.go

```diff
@@ -19,15 +19,22 @@ package server
 import (
 	"fmt"
 	"os"
+	"time"
 
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/errdefs"
 	"github.com/cri-o/ocicni/pkg/ocicni"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
 	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+
+	sandboxstore "github.com/containerd/cri-containerd/pkg/store/sandbox"
 )
 
+// stopCheckPollInterval is the the interval to check whether a sandbox
+// is stopped successfully.
+const stopCheckPollInterval = 100 * time.Millisecond
```

Comment:
todo make configurable

Reply:
We'll remove this very soon by adding a stop channel. @yanxuean @miaoyq are going to do that.
See https://github.com/containerd/cri-containerd/pull/570